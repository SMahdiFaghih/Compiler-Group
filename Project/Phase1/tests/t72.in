23 61		  82 91
		 23 421 5123 12
23