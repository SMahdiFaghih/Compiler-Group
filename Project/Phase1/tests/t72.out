T_INTLITERAL 23
T_INTLITERAL 61
T_INTLITERAL 82
T_INTLITERAL 91
T_INTLITERAL 23
T_INTLITERAL 421
T_INTLITERAL 5123
T_INTLITERAL 12
T_INTLITERAL 23